### ISSUE TITLE
How this mod functions

### ISSUE THREAD

**Issue by CreepyManBun** [ramidzkh/Applied-Botanics#34]
There should be some tutorial or achievement system added to teach people how to use these mechanics. I got this mod for my modpack with buddies because my main problem with botania long distance mana travel but there is nothing in game to teach you how to use the stuff of this addon and no video anywhere that I can find explains how to, not even the videos linked by the mod itself!

**Comment by ramidzkh** [comment]
true, but i have no experience making instructional videos so im just hoping someone else wants to make it instead :shrug:

**Comment by notliam1** [comment]
> true, but i have no experience making instructional videos so im just hoping someone else wants to make it instead 🤷

Why not try to study patchouli and make an extension to the lexica if its possible
something like TC6 addons.

**Comment by Pultex** [comment]
You have to make a mana cell like any other cell
place that in a drive
place a fluix pool down connected to your AE system
aim your spreaders at the fluix pool to insert mana into the mana cell
if you want it to go faster place a spark on the fluix pool and sparks on the normal botanai pools with recessive augment 
faster even more add the dominance augment to the spark with more pools from botania filled with mana and normal sparks on them

each cell holds a lot of mana make sure if you want to use the normal botania system you have a separate mana system

The video the creator made shows you how to transfer mana to other botania devices

the idea is once it is in the system, ae 2 lets you select the mana with an exporter
by right clicking with a mana tablet or ring of mana in the gui to select mana as the export item
have the exporter facing the botania device like the altar

the other way is to  use the p2p system and have the mana move directly to the botania device instantly to infuse items with mana 

this even through works dimensions as AE 2 can use the quantum ring to connect anywhere

**Comment by valerian** [comment]
What is the purpose of the portable mana cell? it doesn't seem to act like a mana tablet
